README.md

```diff
@@ -1,69 +1,69 @@
-[![License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat)](LICENSE)
-[![Dependency Status](https://david-dm.org/silviomoreto/bootstrap-select.svg)](https://david-dm.org/silviomoreto/bootstrap-select)
-[![devDependency Status](https://david-dm.org/silviomoreto/bootstrap-select/dev-status.svg)](https://david-dm.org/silviomoreto/bootstrap-select#info=devDependencies)
-
-A custom select / multiselect for Bootstrap using button dropdown, designed to behave like regular Bootstrap selects.
-
-## Demo and Documentation
-
-See a Bootstrap 3 example [here](http://silviomoreto.github.io/bootstrap-select).
-
-## Authors
-
-[Silvio Moreto](https://github.com/silviomoreto),
-[Ana Carolina](https://github.com/anacarolinats),
-[caseyjhol](https://github.com/caseyjhol),
-[Matt Bryson](https://github.com/mattbryson), and
-[t0xicCode](https://github.com/t0xicCode).
-
-## Usage
-
-Create your `<select>` with the `.selectpicker` class.
-```html
-<select class="selectpicker">
-  <option>Mustard</option>
-  <option>Ketchup</option>
-  <option>Barbecue</option>
-</select>
-```
-
-If you use a 1.6.3 or newer, you don't need to to anything else, as the data-api automatically picks up the `<select>`s with the `selectpicker` class.
-
-If you use an older version, you need to add the following either at the bottom of the page (after the last selectpicker), or in a [`$(document).ready()`](http://api.jquery.com/ready/) block.
-```js
-// To style only <select>s with the selectpicker class
-$('.selectpicker').selectpicker();
-```
-Or
-```js
-// To style all <select>s
-$('select').selectpicker();
-```
-
-Checkout the [documentation](http://silviomoreto.github.io/bootstrap-select) for further information.
-
-## CDN
-
-**N.B.**: The CDN is updated after the release is made public, which means that there is a delay between the publishing of a release and its availability on the CDN. Check [the GitHub page](https://github.com/silviomoreto/bootstrap-select/releases) for the latest release.
-
-* [//cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/css/bootstrap-select.min.css](//cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/css/bootstrap-select.min.css)
-* [//cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/js/bootstrap-select.min.js](//cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/js/bootstrap-select.min.js)
-* //cdnjs.cloudflare.com/ajax/libs/bootstrap-select/1.6.3/js/i18n/defaults-*.min.js (The translation files)
-
-## Bugs and feature requests
-
-Anyone and everyone is welcome to contribute. Please take a moment to
-review the [guidelines for contributing](CONTRIBUTING.md). Make sure you're using the latest version of bootstrap-select before submitting an issue.
-
-* [Bug reports](CONTRIBUTING.md#bug-reports)
-* [Feature requests](CONTRIBUTING.md#feature-requests)
-
-## Copyright and license
-
-Copyright (C) 2013-2015 bootstrap-select
-
-Licensed under [the MIT license](LICENSE).
+bootstrap-select
```

Comment:
Looks like I inadvertently changed the line endings changed from `\r\n` to `\n`. No changes were really made to this README.